index.html

```diff
@@ -0,0 +1,196 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+<!-- The Links --> 
+<link rel="stylesheet" href="style.css">
+<link rel="stylesheet" href="font-awesome/css/font-awesome.css">
+<link
+      href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter:wght@400;600&family=Poppins&family=Roboto&display=swap"
+      rel="stylesheet"
+    /> 
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+</head>
+<body>
+<!-- The Header -->   
+  <header class="header">  
+   <a href="#" class="logo">Gulalai Company </a>  
+    <button type="button"> <i class="fa fa-bars"></i>
+    </button> 
+  </header>
+  <section class="explain container">
+    <h1>Hey There.
+      I’m Gulalai</h1> 
+    <h3>I’am a Software Developer</h3>
+    <h5> I can help you build a product , 
+      feature or website Look through some of my work and experience! 
+      If you like what you see and have a project you need coded, 
+      don’t hestiate to contact me.</h5> 
+      <div class="contact ">
+        <ul class="container">
+          
+          <li><a href="#"><i class="fa fa-linkedin" ></i> </a></li>
+          <li><a href="#"><i class="fa fa-twitter" > </i></a></li>
+          <li><a href="#"><i class="fa fa-github" > </i></a></li>
+          <li><a href="#"><i class="fa fa-angellist" > </i></a></li>
+          <li><a href="#"><i class="fa fa-medium" > </i></a></li> 
+      </ul>
+      </div>
+      <img src="images/Group.png" class="img-fluid ${3|rounded-top,rounded-right,rounded-bottom,rounded-left,rounded-circle,|}" alt="">
+  </section> 
+  <div class="works row">
+    <h1 class="title">My Recent Works</h1>
+    <hr>
+     <div class="image col-11"></div>
+     <h2 class="corner-title container">Multi-Post Stories</h2>
+     <p class="content container">A daily selection of privately personalized
+        reads; no accounts or sign-ups required.
+       has been the industry's standard dummy
+       text ever since the 1500s, when an 
+       unknown printer took a standard dummy text.</p><br>
+     <div class="lessons container "> 
+      <span class="css">css</span>
+     <span class="html">html</span> 
+     <span class="bootstrap boot1">bootstrap</span>
+     <span class="ruby ">Ruby</span>
```

Comment:
Please use the form lists tag. it's good positioned but using lists is better for your code clearness.